data/partials/requests.yaml

```diff
@@ -1,49 +1,54 @@
 requests:
   - name: "OpenMetrics"
-    description: "The Agent includes OpenMetrics and Prometheus checks capable of scraping Prometheus endpoints."
-    link: "/agent/docker/prometheus"
+    description: "The Agent includes the OpenMetrics check capable of scraping Prometheus endpoints. Metrics retrieved by this integration are considered [custom metrics](https://docs.datadoghq.com/metrics/custom_metrics/)."
+    link: "/integrations/openmetrics/"
     image: "int_openmetrics.png"
-  - name: "Windows Performance Counters"
-    description: "Windows services or applications that expose performance counters can be collected with the PDH integration."
-    link: "/integrations/pdh_check"
-    image: "int_wpc.png"
   - name: "JMX Beans"
-    description: "JVM-based applications expose JMX metrics that can be collected with the JMX integration. For example, the JMX integration is already used for official integrations like Solr, Tomcat, Cassandra, and more."
+    description: "The JMX integration enables collection of metrics, logs, and traces from JVM-based applications. For example, the JMX integration is already used for official integrations like Solr, Tomcat, Cassandra, and more. Metrics generated through JMX-based integrations not natively supported by Datadog are considered [custom metrics](https://docs.datadoghq.com/metrics/custom_metrics/)."
     link: "/integrations/java"
     image: "int_jmx.png"
+  - name: "Custom Metrics and Integrations"
+    description: "Submit custom metrics for business stats using [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/) and the [API](https://docs.datadoghq.com/api/latest/metrics/). Datadog Agent integrations are Python files querying for metrics. All Agent code is open source, so it's possible to write your own [custom Agent check](https://docs.datadoghq.com/developers/custom_checks/) or [custom Agent integration](https://docs.datadoghq.com/developers/integrations/). The [integrations-extras GitHub repository](https://github.com/DataDog/integrations-extras) contains many community developed custom Agent integrations."
```

Comment:
Note - Our integrations-extras repo contains both community developed Agent as well as crawler-based integrations. Suggested copy below for the last sentence:

The [integrations-extras GitHub repository](https://github.com/DataDog/integrations-extras) contains many community developed custom integrations."